pkg/kn/traffic/compute.go

```diff
@@ -276,8 +279,17 @@ func Compute(cmd *cobra.Command, targets []servingv1.TrafficTarget, trafficFlags
 	traffic := newServiceTraffic(targets)
 
 	// First precedence: Untag revisions
+	var errTagNames []string
 	for _, tag := range trafficFlags.UntagRevisions {
-		traffic.untagRevision(tag)
+		tagName := traffic.untagRevision(tag, serviceName)
+		if tagName != "" {
+			errTagNames = append(errTagNames, tagName)
+		}
+	}
+
+	// Return all errors from untagging revisions
+	if len(errTagNames) > 0 {
+		return nil, fmt.Errorf("tag(s) %s not present for any revisions of service %s", strings.Join(errTagNames, ", "), serviceName)
```

Comment:
Updated formatting of error message to include all invalid tags used with `--untag` for a particular service and including name for more context with the error message.